plugins/source/aws/resources/services/kafka/clusters_mock_test.go

```diff
@@ -38,5 +38,7 @@ func buildKafkaClustersMock(t *testing.T, ctrl *gomock.Controller) client.Servic
 	}
 }
 func TestKafkaClusters(t *testing.T) {
-	client.AwsMockTestHelper(t, Clusters(), buildKafkaClustersMock, client.TestOptions{})
+	client.AwsMockTestHelper(t, Clusters(), buildKafkaClustersMock, client.TestOptions{
+		SkipEmptyCheckColumns: map[string][]string{"aws_kafka_cluster_operations": {"tags"}},
```

Comment:
Here `tags` field is redundant and is always empty. The resolver ignores this but the field ends up empty.